cosmos/kaiyo.json

```diff
@@ -60,6 +60,12 @@
       "coinGeckoId": "nami-protocol",
       "coinImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/kaiyo/factory/kujira13x2l25mpkhwnwcwdzzd34cr8fyht9jlj7xu9g4uffe36g3fmln8qkvm3qn/unami.png"
     },
+    {
+      "coinDenom": "naUSD",
+      "coinMinimalDenom": "factory/kujira18thffdpuz8tm3mwwun4uhks8a2v7ye0jmw3fllqhklma82lqkpms20gh9c/unausd",
+      "coinDecimals": 6,
+      "coinImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/kaiyo/factory/kujira18thffkujira18thffdpuz8tm3mwwun4uhks8a2v7ye0jmw3fllqhklma82lqkpms20gh9c/unausd.png"
```

Comment:
The image url is incorrect! Please take a look on it

Reply:
Hi, sorry for the inconvenience. Fixed it